python/nwsc_dummy_service/ncd_web_service.py

```diff
@@ -0,0 +1,145 @@
+"""NWS Connect Dummy service simulating behaviors of NWS Connect core services"""
+# ----------------------------------------------------------------------------------
+# Created on Fri Apr 07 2023
+#
+# Copyright (c) 2023 Colorado State University. All rights reserved.             (1)
+#
+# Contributors:
+#     Paul Hamer (1)
+#     Mackenzie Grimes (1)
+#
+# ----------------------------------------------------------------------------------
+import os
+from datetime import datetime, UTC
+from argparse import ArgumentParser, Namespace
+
+from flask import Flask, current_app, request, jsonify
+
+from src.profile_store import ProfileStore
+
+# constants
+GSL_KEY = '8209c979-e3de-402e-a1f5-556d650ab889'
+
+
+def to_iso(date_time: datetime) -> str:
```

Comment:
Does any part of idsse-testing us commons?

Reply:
It doesn't at the moment. I was afraid to add an import of idss-engine-commons just for `to_iso()` and potentially cause a circular dependency in GitHub, because the `idsse/testing` directory has an `idsse_common` folder with test files, which I thought meant commons imports _this_ repo.

Reply:
idsse/testing is a package install and other than a small amount of code that imports resources for tests there is no dependency.